tools/abel/README.md

```diff
@@ -0,0 +1,127 @@
+# ABEL
+### Automated Building Entity Loader
+
+## Overview
+ABEL is a tool allowing system integrators to easily and efficiently create and modify [Building Configuration files](../../ontology/docs/building_config.md).
+
+## Setup and installation
```

Comment:
nit: write something here it is a bit weird to have a subtitle then immediately a sub-subtitle.

Reply:
Added a few sentences, but it feels slightly out of place.

Reply:
LGTM, I'll let @charbull resolve conversation